packages/react-redux-subspace/package.json

```diff
@@ -9,13 +9,16 @@
   ],
   "license": "BSD-3-Clause",
   "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
+  "module": "lib/index.es.js",
```

Comment:
Is there any advantage to moving the the es module into a `es` directory? 

e.g.
```json
"module": "es/index.js"
```

Then someone could force the use of the es module like so:
```js
import { SubspaceProvider } from 'redux-subspace/es'
``` 

At least, I think that's how it works?

Reply:
yeah, it would work like that (same can be done now, but with more verbose - `redux-subspace/lib/index.es.js`)

I dont know why anyone would like to force this, tools should be responsible for making the best choice based on their capabilities

Reply:
If there is no difference in it then I'd prefer the `ex/index.js` approach.

Reply:
ok, I've adjusted the PR